README.md

```diff
@@ -43,6 +43,13 @@ Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](https://github.com/kennyegun24/Mobile-Version/issues).
 
+ **Contributor**
+
+👤 **John Adibe**
+- GitHub: [@githubhandle](https://github.com/johnadibe)
+- Twitter: [@twitterhandle](https://twitter.com/JohnAdibe2)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/john-adibe-400b36166/)
+
 ## Show your support
 
 Give a ⭐️ if you like this project!
```

Comment:
- [ ] the MIT.md link in your README is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉 Also, I suggest removing the last two lines 👌

![0](https://user-images.githubusercontent.com/45231276/188539823-e249a2e3-3048-4b1d-b3f9-9226738f4121.PNG)

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

Reply:
Hello good morning
Can I add some restrictions instead of making it all open?

Reply:
Hi @kennyegun24 
Yes, It's up to you

Reply:
Thanks, I have removed the irrelevant lines and added the required file